content/en/kcd-helsinki-2026/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: KCD Helsinki 2026
+linkTitle: KCD Helsinki 2026
+toc_hide: false
+list_pages: false
+menu:
+  main:
+    weight: 5
+description: 
+---
+
+## Save the date!
+
+KCD Helsinki 2026 will be organized on **May 20, 2026** in **Aalto Dipoli**.
+
+On 20th of May, 2026 the Kubernetes and Cloud Native Community will gather in Helsinki for a one day technical
+conference full of exciting talks and plenty of opportunities for growing and networking. This technical conference will
+attract hundreds of engineers, IT professionals, architects and cloud native stakeholders who come from the local cloud
+native community as well as international speakers and attendees.
+
+## Sponsorship opportunity
+
+We are thrilled to present you with the unique opportunity to sponsor KCD Helsinki 2026, an extraordinary event that
+plays a vital role in fostering the local and global community of engineers, developers, and cloud-native enthusiasts.
+As the adoption of Kubernetes and cloud-native technologies continues to surge, KCDs have emerged as one of the premier
+platforms for connecting like-minded professionals who shape the future of digital infrastructure. KCD Helsinki is the
+connection from Helsinki to the global network of KCDs and cloud-native community.  
+
+Check our [sponsorship prospectus](https://www.canva.com/design/DAGyGVxjIwg/o3qiyrPTE6fA1fFA4srNjw/view?utm_content=DAGyGVxjIwg&utm_campaign=designshare&utm_medium=link2&utm_source=uniquelinks&utlId=hbc4247c13a)!
+
+Contact [sponsor at cloudnativefinland.org](mailto:sponsor@cloudnativefinland.org) in case of any querries.   
```

Comment:
```suggestion
Contact [sponsor at cloudnativefinland.org](mailto:sponsor@cloudnativefinland.org) in case of any queries.   
```

Reply:
Shall we also mention that this email is for sponsor queries and give the kcd Organizer email alias for any other general queries

Reply:
Added in [53f0758](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/21/commits/53f07585c3985a9e74fc48a554a7ef23ee993ca9)